<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemWidget</class>
 <widget class="QWidget" name="ItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>159</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ICEGUI_NS::EditedGroupBox" name="maskGB">
     <property name="title">
      <string>Эталон</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLabel" name="erodeMaskL">
          <property name="text">
           <string>Коэффициент рассеяния:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QSpinBox" name="erodeMaskSB">
          <property name="value">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="dilateMaskL">
          <property name="text">
           <string>Коэффициент расширения:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="dilateMaskSB">
          <property name="value">
           <number>12</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="minAreaL">
          <property name="text">
           <string>Минимальная площадь:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QSpinBox" name="minAreaSB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="minPerimeterL">
          <property name="text">
           <string>Минимальный периметр:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="minPerimeterSB">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="minAreaChB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="minPerimeterChB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="minKFormChB">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="minKFormL">
          <property name="text">
           <string>Коэффициент формы:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QDoubleSpinBox" name="minKFormDSB">
          <property name="minimum">
           <double>1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>10000.000000000000000</double>
          </property>
          <property name="value">
           <double>2.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="erodeFirstMaskChB">
          <property name="text">
           <string>Сначала рассеяние</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="ICEGUI_NS::LineViewWidget" name="lineViewWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="ICEGUI_NS::MaskViewWidget" name="maskViewWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ICEGUI_NS::MaskViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">icegui/maskviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICEGUI_NS::LineViewWidget</class>
   <extends>QWidget</extends>
   <header location="global">icegui/lineviewwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ICEGUI_NS::EditedGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">icegui/editedgroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icefindobjectproc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minAreaChB</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ItemWidget</receiver>
   <slot>minAreaChB_stateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minPerimeterChB</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ItemWidget</receiver>
   <slot>minPerimeterChB_stateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minKFormChB</sender>
   <signal>stateChanged(int)</signal>
   <receiver>ItemWidget</receiver>
   <slot>minKFormChB_stateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>minAreaChB_stateChanged(int)</slot>
  <slot>minPerimeterChB_stateChanged(int)</slot>
  <slot>minKFormChB_stateChanged(int)</slot>
 </slots>
</ui>
